src/main/kotlin/io/github/yuutoproject/yuutobot/commands/minigame/MinigameInstance.kt

```diff
@@ -0,0 +1,182 @@
+/*
+ * Open source bot built by and for the Camp Buddy Discord Fan Server.
+ *     Copyright (C) 2020  Kyuuto-devs
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.yuutoproject.yuutobot.commands.minigame
+
+import io.github.yuutoproject.yuutobot.commands.Minigame
+import io.github.yuutoproject.yuutobot.objects.Question
+import net.dv8tion.jda.api.EmbedBuilder
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionAddEvent
+import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionRemoveEvent
+import java.awt.Color
+
+class MinigameInstance(
+    private val questions: MutableList<Question>,
+    // The ID of the channel the game is in
+    // Effectively the ID of the game instance itself
+    val id: Long,
+    // This is necessary for when the game is finished and we want to remove the object
+    private val manager: Minigame,
+    private val maxRounds: Int
+) {
+    // Is the game in progress or not?
+    var begun = false
+
+    // String ID of user and their scores
+    var players = mutableMapOf<Long, Int>()
+    var timer = System.currentTimeMillis()
+
+    private var rounds = 1
+
+    private lateinit var startingMessageID: String
+
+    private lateinit var currentQuestion: Question
+    private lateinit var currentAnswers: MutableList<String>
+
+    fun start(event: GuildMessageReceivedEvent) {
+        event.channel.sendMessage("Starting a game with $maxRounds rounds...").queue()
+
+        val embed = EmbedBuilder()
+            .setColor(Color.BLUE)
```

Comment:
```suggestion
            .setColor(0xFF93CE)
```
Use Yuuto Pink for all embeds

Reply:
Also stay away from the color class, it's old and poorly optimized.
Actually, just stay away from the entire `java.awt` package